app/networkapi/settings.py

```diff
@@ -203,6 +202,15 @@
     'mezzanine.generic.models.ThreadedComment',
 ]
 
+ADMIN_MENU_ORDER = (
+    ('Content', ('pages.Page', 'blog.BlogPost', 'generic.ThreadedComment',
+                 ('Media Library', 'media-library'))),
+    ('Site', ('sites.Site', 'redirects.Redirect', 'conf.Setting')),
```

Comment:
remove conf.settings? Either that, or we need to make sure that only users with admin rights can see that section (as a followup)